inference-engine/src/transformations/src/ngraph_ops/proposal_ie.cpp

```diff
@@ -48,8 +47,12 @@ void op::ProposalIE::validate_and_infer_types() {
 
         auto batch_size = class_probs_shape[0];
         set_output_type(0, get_input_element_type(0), Shape {batch_size * m_attrs.post_nms_topn, 5});
+        if (get_output_size() == 2)
```

Comment:
In nGraph we usually set output size during `validate_and_infer_types` method. I suggest to parameterize ProposalIE with  additional attribute like `add_second_output`

Reply:
Agreed. Now ProposalAttrs have additional attribute `infer_probs`. Please review.